package/gluon-respondd/files/etc/init.d/gluon-respondd

```diff
@@ -7,13 +7,16 @@ START=50
 SERVICE_WRITE_PID=1
 SERVICE_DAEMONIZE=1
 
-DEVLIST=/var/run/gluon-respondd.devs
 DAEMON=/usr/bin/respondd
 LOCK=/var/run/gluon-respondd.lock
 
 
 do_start() {
-	DEVS=$(cat $DEVLIST 2>/dev/null | while read dev iface; do echo -n " -i $dev"; done)
+	DEVS=""
+	for dev in $( ubus call network.interface dump | jsonfilter -e "@.interface[@.proto='gluon_mesh' && @.up=true].device") br-wan br-client
```

Comment:
actually having br-wan in there is just to copy the current state. It is there because I was not brave enough to remove it on my own. If you feel it needs to go then I will remove it.